File: python/transmit/resources/sandbox.py
from typing import Any, Dict, List


class Sandbox:
    def __init__(self, client):
        self._client = client

    def create_data(self, data: Dict[str, Any]) -> Dict[str, Any]:
        """Create sandbox test data"""
        return self._client.post("/api/v1/developers/sandbox", json=data)

    def list_data(self) -> List[Dict[str, Any]]:
        """List sandbox data"""
        return self._client.get("/api/v1/developers/sandbox")

    def clear_data(self) -> None:
        """Clear all sandbox data"""
        self._client.delete("/api/v1/developers/sandbox")

File: go/transmit/sandbox.go
package transmit

import (
	"encoding/json"
)

type SandboxService struct {
	client *Client
}

func (s *SandboxService) CreateData(req map[string]interface{}) (map[string]interface{}, error) {
	resp, err := s.client.sendRequest("POST", "/api/v1/developers/sandbox", req)
	if err != nil {
		return nil, err
	}

	var result map[string]interface{}
	err = json.Unmarshal(resp, &result)
	return result, err
}

func (s *SandboxService) ClearData() error {
	_, err := s.client.sendRequest("DELETE", "/api/v1/developers/sandbox", nil)
	return err
}
